recipes/hiredis/0.14.1/conanfile.py

```diff
@@ -0,0 +1,78 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+
+from os import rename
+from os.path import join
+
+
+class HiredisConan(ConanFile):
+    name = "hiredis"
+
+    description = "Minimalistic C client for Redis >= 1.2"
+    topics = "conan", "c", "redis"
+
+    license = "BSD 3-Clause \"New\" or \"Revised\" License"
+    settings = "os", "arch", "compiler", "build_type"
+
+    homepage = "https://github.com/redis/hiredis"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False]
+    }
+    default_options = {
+        "fPIC": True,
+        "shared": True
+    }
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_folder = self.name + "-" + self.version
+        rename(extracted_folder, "src")
+
+    def build(self):
+        # settings
+        isWindows = self.settings.os == "Windows"
```

Comment:
`self.settings.os` means the os of the host system (=the system you're builing the libary for).
Here, you want to check the build system (=the system you're building the library on)
```suggestion
        isWindows = tools.os_info.is_windows
```